tests/component/encapsulation.cpp

```diff
@@ -0,0 +1,31 @@
+#include "gtest/gtest.h"
+
+#include <libcellml>
+
+
+TEST(Encapsulation, serialise) {
+    libcellml::Component parent = libcellml::Component();
```

Comment:
For C++ 11 prefer:
auto parent = libcellml::Component();
Incidentally, what you had was equivalent to just:
libcellml::Component parent;
which is even less clear that a new component is being made.

If you follow my other recommendations to return std::shared_ptr and have a static create method this would become:
auto parent = libcellml::Component::create();
which is pretty clear about what it does; the auto type will then be a shared_ptr.